integration-tests/test_connection.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+import conftest
+
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_connection(insights_client):
+    """
+    Test insights-client --test-connection
+    `test_connection` method returns 0 if success, 1 if failure
+    """
+    url_test = "End Upload URL Connection Test: SUCCESS"
+    api_test = "End API URL Connection Test: SUCCESS"
+
+    insights_client.register()
```

Comment:
Do we really need to register for this test?? I mean subman is register so certs would be available for connection test. In my opinion you could skip this step.